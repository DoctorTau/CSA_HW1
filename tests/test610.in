450
-1792 -159 3609 2499 -2100 -2215 4956 7290 8171 -6913 -5843 -1136 -3678 -6857 -9906 5971 -7617 -5298 -4511 -4820 4296 2702 9898 -5053 -7055 -5763 -6839 -689 -5871 -3976 -6678 3759 1111 5299 -731 -9845 -1536 5571 -7539 7476 -1958 1326 2094 1779 5993 -4243 8142 9174 1882 2760 -2823 -4418 4023 6724 -7979 -1946 8982 -7705 6797 -7820 -9315 -1268 -8219 2350 -9102 -77 942 9523 -6400 -4172 9052 8517 -9167 2379 -6733 7168 -2290 1966 -9855 -1213 -3571 -8889 -5008 8746 2736 1534 2445 -4824 -7774 -6830 -7610 9745 1434 -2782 4652 9333 3271 -333 4213 8929 1790 5655 -670 -433 -9950 7358 -9235 1630 -2651 8651 3816 487 1276 9396 9612 2706 -7567 -7119 2267 -750 5535 5716 6513 -9859 2450 7250 -6679 1028 8876 5860 -1715 6420 -9177 -8250 -8011 9321 9207 -734 -1154 -3952 6265 -3684 2265 926 -3260 7224 -6877 -8986 -9972 -4249 5034 -3495 -9744 2516 8963 6842 694 -2990 -2947 5236 -9009 -8613 -901 -9778 -6722 -8733 -564 -9972 -9129 2050 314 -7106 1827 -6243 -969 956 1514 1620 7504 8337 -3139 187 6520 2795 -6332 -845 6904 -7918 -2643 2277 3078 6391 3150 -3330 2472 -5171 -2937 7297 -5342 1510 7620 -9004 -9742 2005 -9963 -9658 -3098 5908 6425 -6386 -5995 -9172 -2065 105 -8459 4849 -5719 9005 9844 9584 3999 -1856 4998 -6350 5751 382 -2574 -5441 2315 -3286 -7391 3825 -8979 663 -7073 4324 6042 -6958 1440 -34 -806 -2863 84 9261 -4743 9812 -305 3064 3747 4328 1687 3522 1457 -2056 135 2957 9177 1232 -810 -9847 2627 -3170 4478 -4979 -758 5703 3524 8342 -6486 8130 -2276 -1011 9668 -1432 -5108 2327 3677 7990 5901 558 -9988 -3259 -5303 -1185 -6529 5019 8904 -3713 -2835 1834 -6851 5634 6629 3595 1730 -3511 -7159 -8670 -674 4813 9998 7157 -847 9239 -2306 6985 -7165 -9822 6115 -1960 432 2423 -9164 7776 8583 8945 -1852 -2744 9457 4295 -928 -9505 295 -5691 607 8376 -9181 7698 8785 -600 5287 5486 6459 8759 4680 9153 -841 5253 -1533 647 -7338 7081 7551 -912 -1160 6152 -8285 1656 7385 -7856 -8529 7198 8179 -2171 5208 1696 -4602 -6717 9645 5564 -2375 -6262 -1557 -5243 -8979 131 -8587 9742 8799 -6685 7407 -9499 8619 -257 -158 -6263 4999 427 5498 2137 -6129 -8508 -1536 6147 5231 -7149 -4137 -1053 306 -1454 4816 7864 7742 8718 -5652 7657 9133 6552 -5440 3314 -5991 -9353 -3930 -494 -9058 -1551 -4623 -753 9924 -7960 2356 1545 -4711 6521 5674 -1822 7303 3701 2201 -5826 5353 -4972 -7018 -8561 4370 65 -6280 2481 -9818 -8601 767 2525 -6374 -3553 3852 -1001 3217 -404 -4780 4767 8517 -6567 -1538 1312 -354 -4545 2307 1853 -6098 -3416 