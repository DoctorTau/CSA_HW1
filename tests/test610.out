-1792 -2100 -1136 -3678 -9906 -5298 -4820 -3976 -6678 -1536 -1958 -4418 -1946 -7820 -1268 -9102 -6400 -4172 -2290 -5008 -4824 -7774 -6830 -7610 -2782 -670 -9950 -750 -8250 -734 -1154 -3952 -3684 -3260 -8986 -9972 -9744 -2990 -9778 -6722 -564 -9972 -7106 -6332 -7918 -3330 -5342 -9004 -9742 -9658 -3098 -6386 -9172 -1856 -6350 -2574 -3286 -6958 -34 -806 -2056 -810 -3170 -758 -6486 -2276 -1432 -5108 -9988 -8670 -674 -2306 -9822 -1960 -9164 -1852 -2744 -928 -600 -7338 -912 -1160 -7856 -4602 -6262 -158 -8508 -1536 -1454 -5652 -5440 -3930 -494 -9058 -7960 -1822 -5826 -4972 -7018 -6280 -9818 -6374 -404 -4780 -1538 -354 -6098 -3416 